src/pages/GettingStarted.js

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { H1, H2, H3, P, A, UL, LI } from '../components/Typography'
+import { Checkbox } from '../components/Input'
+
+const StyledH2 = styled(H2)`
+  opacity: 1;
+`
+const StyledH3 = styled(H3)`
+  opacity: 1;
+`
+const StyledP = styled(P)`
+  margin-top: 5px;
+`
+const TodoColumnContainer = styled.div`
```

Comment:
Mobile view is a bit cramped -- maybe a media query w/ mobile breakpoint that adds flex-direction: column;?

Reply:
Talked to Anson and there's some style nits that I'm going to take care of. I'll address this one in those changes as well!